packages/react-sdk/src/useSpace.ts

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import { useContext, useMemo } from 'react'
+import { useSyncAgent } from './useSyncAgent'
+import { SpaceContext } from './internals/SpaceContext'
+
+export const useSpace = (spaceId: string) => {
+    const sync = useSyncAgent()
+    const space = useMemo(() => sync.spaces.getSpace(spaceId), [sync, spaceId])
+    return space
```

Comment:
This should be return useObservable(space) right?

Reply:
Yeah now I'm seeing this piece of code with different eyes -- This should wrap the space in a useObservable, any function like `createChannel` and so on will be done using a different hook, like `useCreateChannel`/ `useRiverAction`